evaluation.py

```diff
@@ -97,7 +98,40 @@ def save_stacked_evaluation_heatmaps(self, h, X, y, stacked_predict_heatmaps_fil
         img_v_resize = self._vstack_images(heatmap_imgs) 
         
         cv2.imwrite(filename, img_v_resize) 
-    
+
+    # Resources for heatmaps to keypoints
+    # https://github.com/yuanyuanli85/Stacked_Hourglass_Network_Keras/blob/eddf0ae15715a88d7859847cfff5f5092b260ae1/src/eval/heatmap_process.py#L5
+    # https://github.com/david8862/tf-keras-stacked-hourglass-keypoint-detection/blob/56707252501c73b2bf2aac8fff3e22760fd47dca/hourglass/postprocess.py#L17
+   
+    ### Returns np array of predicted keypoints from one image's heatmaps
+    def heatmaps_to_keypoints(self, heatmaps, threshold=HM_TO_KP_THRESHOLD):
+        keypoints = list()
+        for i in range(NUM_COCO_KEYPOINTS):
+            hmap = heatmaps[:,:,i]
+            # Resize heatmap from Output DIM to Input DIM
+            resized_hmap = cv2.resize(hmap, INPUT_DIM, interpolation = cv2.INTER_AREA)
```

Comment:
According to [this](https://stackoverflow.com/questions/23853632/which-kind-of-interpolation-best-for-resizing-image), when enlarging an image, we should use `INTER_LINEAR` (computationally faster but less quality) OR `INTER_CUBIC` (computationally slower but higher quality). `INTER_LINEAR` is default for cv2 so we may wanna just roll with this. I've made a note to go back to the data generator and apply `INTER_AREA` which is better for shrinking images.

Reply:
Good catch!